bugbug/models/invalid_compatibility_report.py

```diff
@@ -50,7 +50,8 @@ def __init__(self, lemmatization=False):
             ]
         )
 
-        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+        self.hyperparameter = {"n_jobs": utils.get_physical_cpu_count()}
+        self.clf = xgboost.XGBClassifier(**self.hyperparameter)
 
```

Comment:
(noticed this was added to all models and the training task fails without it)

Reply:
Could you please submit it in a separate PR?

Reply:
Yeah, sounds good!